apps/condo/domains/resident/tasks/sendBillingReceiptsAddedNotificationForOrganizationContextTask.js

```diff
@@ -144,18 +152,28 @@ async function sendBillingReceiptsAddedNotificationForOrganizationContext (conte
 }
 
 async function fetchReceipts (contextId, receiptCreatedAfter) {
-    const periods = [
-        dayjs().subtract(1, 'month').format('YYYY-MM-01'),
-        dayjs().format('YYYY-MM-01'),
-    ]
+    const { keystone: context } = getSchemaCtx('BillingReceipt')
     const receiptsWhere = {
         createdAt_gt: receiptCreatedAfter,
         context: { id: contextId },
-        period_in: periods,
         toPay_gt: '0',
         deletedAt: null,
     }
-    return find('BillingReceipt', receiptsWhere)
+    const receipts = await BillingReceipt.getAll(context, receiptsWhere, BILLING_RECEIPT_FIELDS)
```

Comment:
Will that work?
We have restriction: in one request there can be 1000 entities at max (relations interpreted as entities)

It means 1000 flat receipts, or 500 receipts with BillingIntegrationContext, 333 receipts with context and BillingProperty and so on.

Maybe try to do it chunk by chunk or using flat `find`?